docs/unified-test-documentation/dasharo-security/20S-usb-stack.md

```diff
@@ -0,0 +1,151 @@
+# Dasharo Security: USB stack
```

Comment:
@Pokisiekk, we have some inconsistency here. In our products documentation, we had the product: `USB stack enable/disable`. I think, that in documentation we should use the same nomenclature

Reply:
Renamed